controllers/secretgenerator/secret.go

```diff
@@ -9,18 +9,18 @@ import (
 )
 
 const (
-	SECRET_NAME_ANNOTATION         = "secret-generator.opendatahub.io/name"               //nolint
-	SECRET_TYPE_ANNOTATION         = "secret-generator.opendatahub.io/type"               //nolint
-	SECRET_LENGTH_ANNOTATION       = "secret-generator.opendatahub.io/complexity"         //nolint
-	SECRET_OAUTH_CLIENT_ANNOTATION = "secret-generator.opendatahub.io/oauth-client-route" //nolint
-	SECRET_DEFAULT_COMPLEXITY      = 16                                                   //nolint
+	SECRET_NAME_ANNOTATION         = "secret-generator.opendatahub.io/name"               //nolint:golint,revive,stylecheck //CAPS is preferred for const
```

Comment:
I wonder if you could put //nolint on top of `const (` and it will apply to all the consts below. Otherwise it's quite some duplication. Have you tried that?